package.json

```diff
@@ -38,10 +38,10 @@
     "travisDeployKey": "./bin/generate_travis_deploy_key"
   },
   "dependencies": {
-    "cozy-client": "13.5.2",
+    "cozy-client": "13.6.0",
     "cozy-konnector-libs": "4.34.2",
     "isomorphic-fetch": "2.2.1",
-    "jest": "25.5.4",
+    "jest": "26.0.1",
```

Comment:
Do you really want to upgrade jest? This is not needed by the CozyClient upgrade

Reply:
yes but since the unit tests still work the same way with the new version, I suppose this is OK. Do you see any reason not to do it now ?

Reply:
I like to have one responsibility per commit. Easier to read and understand in a few months when we'll have to debug or work on the connector again ^^

Reply:
Ok, I remove the upgrade of jest since I do not have arguments anymore :-)